assets/js/main.js

```diff
@@ -0,0 +1,240 @@
+/*global
+  $
+  Audio
+*/
+
+(function () {
+  // ================== globals ==================
+  // keycodes for left and right
+  var LEFT = 37;
+  var RIGHT = 39;
+
+  // size of each grid square
+  var GRID_SIZE = 12;
+  // grid height and width
+  var GRID_WIDTH = 50;
+  var GRID_HEIGHT = 40;
+
+  // initial game settings
+  // how fast the game update loop runs
+  var START_TICK = 10;
+  // how many update calls per drop move / collision
+  var START_DROPS_SPEED = 10;
+  // how many update calls per pad movement
+  var START_MOVE_SPEED = 5;
+  // how widet he pad is to start
+  var START_PAD_WIDTH = 4;
+  // how many update calls per createDrop
+  var START_DROP_CREATE_TIME = 99;
+
+  // the jquery game element
+  var $game = null;
+  // the pad that catches the drops
+  var $pad = null;
+  // the jquery score element
+  var $score = null;
+  // all the jquery drop elements
+  var $drops = [];
+
+  // all the game state
+  var game = null;
+
+  // the sound of catching
+  var catchSound = null;
+
+  // ================== view helpers ==================
+  // these are all helpers to get and set model info from a jquery element
+  // the conversion to grid is also handled here
```

Comment:
By using a grid system it makes collision easier and aligns closer with snake should they continue in that direction